intel-retail/automated-self-checkout
# Fix realsense missing parameters from gst-launch command

FIXES: #341

## PR Checklist



## What are you changing?


## Issue this PR will close

close: #**341**

## Anything the reviewer should know when reviewing this PR?

## Test Instructions if applicable

- git clone this PR
- [modify the configuration.yaml of gst profile for DockerImage](https://github.com/intel-retail/automated-self-checkout/blob/main/configs/opencv-ovms/cmd_client/res/gst/configuration.yaml#L20) to be `dlstreamer.realsense`
- make clean-all
- run gst profile with realsense: `PIPELINE_PROFILE=gst sudo -E ./run.sh --workload ovms --platform core --inputsrc your_realsense_serial_number`   and then do a docker log on gst container. and you now should see the color-width and color-height parameters in gst-launch command.  both parameters were missing from gst-launch command before.
- or you could overwrite the COLOR_WIDTH and COLOR_HEIGHT value from the command line like `PIPELINE_PROFILE=gst COLOR_WIDTH=800 COLOR_HEIGHT=640 sudo -E ./run.sh --workload ovms --platform core --inputsrc your_realsense_serial_number` and you should see the docker logs of gst-launch command having those realsense parameter values

## If the there are associated PRs in other repositories, please link them here (i.e. intel-retail/automated-self-checkout )
